src/plugins/footnotes/index.ts

```diff
@@ -146,29 +145,40 @@ const deleteFootnoteWidget =
         if (node.type === schema.nodes.footnote && pos) {
           const targetNode = tableElement ? tableElement.node : view.state.doc
           const inlineFootnotes = getInlineFootnotes(id, targetNode)
-
-          const nodeWithPos = findParentNodeClosestToPos(
+          const footnotesElement = findParentNodeClosestToPos(
+            tr.doc.resolve(pos),
+            (node) => node.type === schema.nodes.footnotes_element
+          )
+          const tableElementFooter = findParentNodeClosestToPos(
             tr.doc.resolve(pos),
-            (node) => node.type === schema.nodes.footnote
+            (node) => node.type === schema.nodes.table_element_footer
```

Comment:
Could you please synchronize your branch with the master? The latest code on master is passing the table_element_footer  node to the deleteFootnoteWidget  and there will be no need for this change.

Reply:
done